releases/release-1.11/release_team.md

```diff
@@ -2,7 +2,7 @@
 | ------ | ------ | ------ |
 | Lead | Josh Berkus ([@jberkus](https://github.com/jberkus))  |  |
 | Features | Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)) |  |
-| CI Signal | | |
+| CI Signal | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | |
```

Comment:
I have a non-binding vote here, but it's unusual if the first-time contributor to sig-release repo is applying for the lead role.

Maybe worth considering the shadow role as a starting point for 1.11 release?